packages/lib/src/Avatar.wc.svelte

```diff
@@ -17,7 +17,7 @@
     <button
       type="button"
       class={twMerge(
-        "relative inline-block rounded-full overflow-hidden bg-gray-200 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-blue-500",
+        "relative inline-block rounded-full overflow-hidden bg-purple-700",
```

Comment:
Changed the default color for the Avatar to purple which for some odd reason seems to be the web standard.